src/database/seeders/20240415080213-demo-user.ts

```diff
@@ -1,56 +1,42 @@
 'use strict';
-const bcrypt = require('bcrypt');
+import dotenv from 'dotenv';
+dotenv.config();
 const { v4: uuid } = require('uuid');
-
 /** @type {import('sequelize-cli').Migration} */
 module.exports = {
-  async up(
-    queryInterface: {
-      bulkInsert: (
-        arg0: string,
-        arg1: {
-          userId: any;
-          firstName: string;
-          lastName: string;
-          email: string;
-          password: any;
-          isVerified: boolean;
-        }[],
-        arg2: {},
-      ) => any;
-    },
-    Sequelize: any,
-  ) {
-    const hashedPassword1 = await bcrypt.hash('Password1239@', 10);
-
+  async up(queryInterface: any, Sequelize: any) {
     await queryInterface.bulkInsert(
       'Users',
       [
         {
           userId: uuid(),
           firstName: 'christian',
           lastName: 'Ishimwe',
-          email: 'iigore2061@gmail.com',
-          password: hashedPassword1,
-          isVerified: true,
+          email: 'christianinja3@gmail.com',
+          password: process.env.password,
```

Comment:
A password should be hashed before being saved to the database, regardless of it being an environment variable